NSS-Day-Cohort-66/rare-api-extra-medium
# correct post test

#Created test for Posts

###Steps to test

pull down branch mjt--postTests 
run this test in your terminal
expected outcome

Found 7 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
.......
----------------------------------------------------------------------
Ran 7 tests in 0.083s

OK
Destroying test database for alias 'default'...
